x-pack/legacy/plugins/apm/public/components/shared/ServiceAlertTrigger/DurationField/index.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { useRef } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFieldNumber,
+  EuiSelect
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import {
+  parseEsInterval,
+  ParsedInterval
+} from '../../../../../../../../../src/legacy/core_plugins/data/common';
+import { getTimeUnitLabel } from '../../../../../../../../plugins/triggers_actions_ui/public';
+import { TIME_UNITS } from '../../../../../../../../plugins/triggers_actions_ui/public';
+
+interface Props {
+  duration: string;
+  onChange: (duration: string) => void;
+}
+
+export function DurationField(props: Props) {
+  const { duration, onChange } = props;
+
+  let parsedInterval: ParsedInterval | undefined;
+
+  try {
+    parsedInterval = parseEsInterval(duration);
+  } catch (err) {
+    //
+  }
```

Comment:
Should the error be handled somehow?

Reply:
It shouldn't, but I'll leave a comment there with the reason.

Reply:
In that case this feels like something that shouldn't be thrown from `parseEsInterval`. But I can see how that is not easily changed.